importers/test/test_djornl_parser.py

```diff
@@ -33,18 +33,29 @@ def init_parser_with_path(self, root_path):
             parser._configure()
             return parser
 
+    def test_load_no_manifest(self):
```

Comment:
:+1: I do admire the work ethic and agree it would be especially useful if we automate this importer. I don't think we will have this high of test standards for the other importers :p